src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public interface Drawable {
```

Comment:
The interface `Drawable` is empty. According to the checklist, interfaces should not contain unrelated methods, but they should contain at least one method that defines the behavior of the implementing classes. Consider adding a method like `void draw();` to define what it means for a class to be drawable.